skill-tree /src/test/java/com/RDS/skilltree/unit/SkillsServiceTest.java

```diff
@@ -0,0 +1,78 @@
+package com.RDS.skilltree.unit;
+
+import com.RDS.skilltree.Skill.*;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SkillsServiceTest {
+
+    @Mock
+    private SkillRepository skillRepository;
+
+    @InjectMocks
+    @Autowired
+    private SkillsServiceImpl skillService;
+
+
+    @Test
+    public void testGetSkillById() {
+        UUID skillId = UUID.randomUUID();
+        SkillModel skillModel = SkillModel.builder().id(skillId).build();
+
+        when(skillRepository.findById(skillId)).thenReturn(Optional.of(skillModel));
+
+        SkillDTO result = skillService.getSkillById(skillId);
+        assertNotNull(result);
+        assertEquals("The skill Id doesn't matches the expected skillId", skillId, result.getId());
```

Comment:
```suggestion
        assertEquals("The skill Id matches the expected skillId", skillId, result.getId());
```

Reply:
> In this case, if the assertion fails, the error message will be printed to the console, helping you identify the issue. If the assertion succeeds, no message is printed unless there is a global setting in your testing environment to display successful assertions (which is not common).

Reply:
@heyrandhir I think this comment we can resolve as this prints in case of asserting being failed